ui/lib/kmip/addon/routes/configure.js

```diff
@@ -17,7 +17,7 @@ export default Route.extend({
     return this.store.findRecord('kmip/config', this.secretMountPath.currentPath).catch((err) => {
       if (err.httpStatus === 404) {
         const model = this.store.createRecord('kmip/config');
-        model.set('id', this.secretMountPath.currentPath);
```

Comment:
resolves this error. i cannot for the life of me figure out where this record is created and/or sent to the server. I tried skipping `.findRecord` but nope, that's not it. My suspicion is that it's something to do with how openApi is generating schemas but I couldn't pin it down
```
router.js:1129 Error while processing route: vault.cluster.secrets.backend.kmip.configure 'kmip/config' was saved to the server, but the response returned the new id 'kmip', which has already been used with another record.' Error: 'kmip/config' was saved to the server, but the response returned the new id 'kmip', which has already been used with another record.'